packages/react-bootstrap-table2-overlay/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-bootstrap-table2-overlay",
-  "version": "1.0.0",
+  "version": "2.0.0",
```

Comment:
hello, i guess you dont rebase the upstream, please rebase it to avoid these useless code change. Thanks